pkgs/tools/misc/less/default.nix

```diff
@@ -8,10 +8,8 @@ stdenv.mkDerivation rec {
     sha256 = "07z43kwbmba2wh3q1gps09l72p8izfagygmqq1izi50s2h51mfvy";
   };
 
-  # Look for ‘sysless’ in /etc.
-  configureFlags = "--sysconfdir=/etc";
-
-  preConfigure = "chmod +x ./configure";
+  configureFlags = "--sysconfdir=/etc " # Look for ‘sysless’ in /etc.
+    + stdenv.lib.optionalString lessSecure "--with-secure ";
```

Comment:
Could you please convert these flags into a list? Specifying a string will be deprecated soon.